src/Screens/Onboarding/MentorList.tsx

```diff
@@ -7,7 +7,7 @@ import * as snapCarousel from 'react-native-snap-carousel';
 import { SafeAreaView } from 'react-navigation';
 
 import * as remoteData from '../../lib/remote-data';
-import useWidth from '../../lib/use-width';
+import useLayout from '../../lib/use-layout';
```

Comment:
I made new useLayout hook that provides {width, height} object and a callback that can be plugged to RN.Views onLayout prop.